### PR TITLE
adding info

### PR DESCRIPTION
**Pull request by Chandan51Coder**

### CODE REVIEW

**Comment by Chandan51Coder** [comment]
adding info

**Comment by Chandan51Coder** [comment]
adding info

File: about.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+<!--      please add some comment to understand the code-->
```
Comment (AmarNathPrajapati) [inline about.html:4]: explain a bit more

**Review by AmarNathPrajapati — COMMENTED** [review]
complete work first

**Comment by AmarNathPrajapati** [comment]
thanks

**Pull request merged**